### DISCUSSION TITLE
#71 辽宁联通节点映射mc服务器连不上，是禁止udp了吗？

### DISCUSSION THREAD

**Discussion by tsdlmx** [natfrp/wiki#103] [使用问题]
#71 辽宁联通节点映射mc服务器连不上，是禁止udp了吗？

**Comment by SanaeAkahane** [comment]
经测试是没有禁止的，且创建隧道页面并没有 `屏蔽UDP` 标识。

但部分地区可能由于网络环境问题，导致出现连不上节点的情况，目前已知的是黑龙江部分地区。
